@jsmartt we could find error message here: https://dev.azure.com/azure-sdk/public/_build/results?buildId=457263&view=logs&jobId=95dbed5b-845d-57ba-33d3-f6c5e69afcf4&j=95dbed5b-845d-57ba-33d3-f6c5e69afcf4&t=44f1510e-a822-56b0-2e0b-241e0a387c6a

which says
```
##[error]Semantically validating  specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2019-11-01/storagecache.json:

 error : 
code: OAV131
error: DISCRIMINATOR_NOT_REQUIRED
name: discriminator
params:
  - targetType
message: discriminator must be a required property.
path:
  - 'definitions,StorageTarget'
```

please check `targetType` field in this file, and mark it as `required`.

--- a/specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2019-11-01/storagecache.json
+++ b/specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2019-11-01/storagecache.json
@@ -1434,7 +1434,6 @@
     "StorageTarget": {
       "description": "A storage system being cached by a Cache.",
       "type": "object",
-      "discriminator": "targetType",
       "properties": {
         "name": {
           "description": "Name of the Storage Target.",
@@ -1455,6 +1454,10 @@
           "x-ms-client-flatten": true,
           "description": "Properties of the Storage Target.",
           "type": "object",
+          "discriminator": "targetType",
+          "required": [
+            "targetType"
+          ],
           "properties": {
             "junctions": {
               "description": "List of Cache namespace junctions to target for namespace associations.",

